app/views/trips/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= @trip.duration %>
```

Comment:
I think this is a good place for a table, and that all three of the indexes we're working on tonight will have similar tables.

Reply:
Yeah I agree. I figured we'd wait on creating a default table structure for all the indexes until we had some sort of wireframe or consensus on how we want it to look. This at least is the data and is pretty easy to add a table to it later.

Reply:
That's a good call. After I finish the second user story, I'll draw up some ideas for wireframes. If we all do the same, we can have a good discussion.